README.md

```diff
@@ -1,141 +1,144 @@
+  "tags": 
     {
-      "name": "city",
-      "value": "new_york"
+      "__name__": "third_avenue",
+      "city": "new_york",
+      "checkout": "1"
     },
-    {
-      "name": "endpoint",
-      "value": "/request"
-    }
-  ],
-  "datapoint": {
-    "timestamp":'"$(date +"%s")"',
-    "value": 42.123456789
-  }
+    "timestamp": '\"$(date "+%s")\"',
+    "value": 3347.26
 }'
 ```
 
-#### Query for reverse indexed time series data
+5.  Query Results
 
+**Linux**
+
+```shell
+curl -X "POST" -G "http://localhost:7201/api/v1/query_range" \
+  -d "query=third_avenue" \
+  -d "start=$(date "+%s" -d "45 seconds ago")" \
+  -d "end=$( date +%s )" \
+  -d "step=5s" | jq .  
 ```
-curl http://localhost:9003/query -s -X POST -d '{
-  "namespace": "default",
-  "query": {
-    "regexp": {
-      "field": "city",
-      "regexp": ".*"
-    }
-  },
-  "rangeStart": 0,
-  "rangeEnd":'"$(date +"%s")"'
-}' | jq .
+
+**macOS/BSD**
+
+```shell
+curl -X "POST" -G "http://localhost:7201/api/v1/query_range" \
+  -d "query=third_avenue > 6000" \
+  -d "start=$(date -v -45S "+%s")" \
+  -d "end=$( date +%s )" \
+  -d "step=5s" | jq .
 ```
 
-## Building with Docker
+## Contributing
 
-A Dockerfile is included for both development and production deployment purposes. It uses a
-[multi-stage build](https://docs.docker.com/develop/develop-images/multistage-build/) in order to
-produce a lightweight production image from a single Dockerfile. Accordingly, it requires Docker
-17.05 or later to build.
+You can ask questions and give feedback in the following ways:
 
-```
-docker build -f docker/m3dbnode/Dockerfile -t m3dbnode:$(git rev-parse head) .
-docker run --name m3dbnode m3dbnode:$(git rev-parse head)
-```
+-   [Create a GitHub issue](https://github.com/m3db/m3/issues)
+-   [In the public M3 Slack](http://bit.ly/m3slack)
+-   [In the M3 forum (Google Group)](https://groups.google.com/forum/#!forum/m3db)
 
-If you wish to build an image with the source code included you can stop the build after the
-`builder` stage:
+M3 welcomes pull requests…
+Read [the development guide](DEVELOPMENT.md) to help you get setup for building and contributing to M3.
 
-```
-docker build -f docker/m3dbnode/Dockerfile -t m3dbnode:$(git rev-parse head) --target builder .
-```
+## Community Meetings
 
-## Configuration
+M3 contributors and maintainers have regular meetings. Join our M3 meetup group to receive notifications on upcoming meetings: 
+<https://www.meetup.com/M3-Community/>.
 
-The default Docker image will start a single `m3dbnode` process with an embedded etcd instance to
-mimic a production environment. If you would like to further customize the configuration, you must
-provide your own and mount it into the container:
+Recordings of past meetups can be found here: <https://vimeo.com/user/120001164/folder/2290331>.
 
-```
-docker run --name m3dbnode -v /host/config.yml:/etc/m3dbnode/myconfig.yml m3dbnode:tag -f /etc/m3dbnode/myconfig.yml
-```
+## Office Hours
```

Comment:
Only thing I would change is to move these three sections to be under "More information".

i.e. headers order should be
1. More information
2. Community meetings
3. Office hours
4. ...everything else...
5. Contributing
6. "This project is released under Apache License, Version 2.0"... etc

This matches the fairly popular as in use with etcd and other projects:
https://github.com/etcd-io/etcd

Reply:
@robskillington We were also following a very popular read me template ;)

https://github.com/RichardLitt/standard-readme

Reply:
but let me check, it's slightly hard to know which sections you're referring to as the diff is so big

Reply:
Though that template doesn't really have a "more information" section

Reply:
OK, think I updated to what you mean, I was mostly concerned it would push the "getting started" steps too far down, but really we want readers to go to the docs anyway